arches/app/media/css/arches.scss

```diff
@@ -2204,10 +2204,6 @@ div.final-cons-step-separator>h4 {
     margin-right: 400px;
 }
 
-.workbench-card-container-wrapper.workbench-card-container-sidepanel-active {
-    z-index: 249;
-}
-
```

Comment:
Removing this prevented a regression with the existing card help panel accessed from the edit tab. This z-index was causing a new stacking context, with the result that the card help panel was appearing under the edit tab despite having a higher z-index.